query.go

```diff
@@ -71,44 +86,72 @@ func (q *dhtQuery) Run(ctx context.Context, peers []peer.ID) (*dhtQueryResult, e
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	runner := newQueryRunner(q)
+	numPaths := NumDisjointPaths
```

Comment:
Let's make this configurable.

Reply:
Sure I'll add an option.

Reply:
Now configurable via `opts.DisjiontPaths`